### PR TITLE
[SJS-65] Allow user to specify max length of a body value

### PR DESCRIPTION
**Pull request by klesgidis**
## Summary

With this PR, we allow user to specify an upper limit for the body value to be logged

### CODE REVIEW

File: lib/transformers/transformers.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+const mapValues = require('lodash/mapValues');
+const isObject = require('lodash/isObject');
 const url = require('url');
 
 const isHealth = require('./isHealth');
 const { extractHeaders, truncateText } = require('../utils');
 
+const mapValuesDeep = (v, callback) => (isObject(v) ? mapValues(v, v => mapValuesDeep(v, callback)) : callback(v));
+
+const transformer = length => v => (typeof v === 'string' && truncateText(v, length, '[Trimmed by riviere]')) || v;
```
Comment (nikostoulas) [inline lib/transformers/transformers.js]: 🙃 rename to sth like `truncateTextWithLength`

File: README.md [hunk 2]
```diff
@@ -24,14 +24,15 @@ Use `riviere` if you want an easy way to log all the HTTP traffic for your serve
...
     5. [bodyKeysRegex](#options_body_keys_regex)
-    6. [color](#options_color)
-    7. [styles](#options_styles)
-    8. [context](#options_context)
-    9. [errors.callback](#options_errors_callback)
-    10. [headersRegex](#options_headers_regex)
-    11. [health](#options_health)
-    12. [outbound.enabled](#options_outbound_enabled)
-    13. [traceHeaderName](#options_trace_header_name)
+    6. [bodyValuesMaxLength] (#options_body_values_max_length)
```
Comment (nikostoulas) [inline README.md]: 🔧 I see that we already support `outbound.maxQueryChars`, `outbound.maxPathChars`, `outbound.maxHrefChars`
So for uniformity I would expect this to be `inbound.maxBodyValuesChars` and `outbound.maxBodyValuesChars`.
Can using only this config ensure a maximum body chars?

Comment (klesgidis) [reply README.md]: Do you suggest splitting this into to options?

It doesn't ensure that the maximum body chars would be under 256K, but I don't expect payloads with many of those fields. I am thinking of putting the limit to something like 10K, so you need to have ~20 files to the payload

**Comment by klesgidis** [comment]
PTAL @nikostoulas

File: README.md (hunk 2 above)
Comment (nikostoulas) [reply README.md]: > Do you suggest splitting this into to options?

Renaming as a first step to match the other configs. If it is not split similarly to other configs won't it be a bit weird as configuration? 
> 
> It doesn't ensure that the maximum body chars would be under 256K, but I don't expect payloads with many of those fields. I am thinking of putting the limit to something like 10K, so you need to have ~20 files to the payload

Ok makes sense

**Review by nikostoulas — APPROVED** [review]

**Pull request merged**